pydantic/_internal/_generate_schema.py

```diff
@@ -486,8 +489,12 @@ def _arbitrary_type_schema(self, obj: Any, type_: Any) -> CoreSchema:
         else:
             raise PydanticSchemaGenerationError(
                 f'Unable to generate pydantic-core schema for {obj!r}. '
-                f'Set `arbitrary_types_allowed=True` in the model_config to ignore this error'
-                f' or implement `__get_pydantic_core_schema__` on your type to fully support it.'
+                'Set `arbitrary_types_allowed=True` in the model_config to ignore this error'
+                ' or implement `__get_pydantic_core_schema__` on your type to fully support it.'
+                'If you got this error by calling handler(<some type>) within'
```

Comment:
```suggestion
                ' If you got this error by calling handler(<some type>) within'
```

Reply:
Also, I think it might be nice to add two newlines before the extra info, since I think needing to add `arbitrary_types_allowed=True` will be far more common than calling `handler(<some type>)`.